MSAL/src/native_auth/network/responses/validator/sign_in/MSALNativeAuthSignInResponseValidator.swift

```diff
@@ -74,10 +74,10 @@ final class MSALNativeAuthSignInResponseValidator: MSALNativeAuthSignInResponseV
             return .error(.unexpectedError(.init(errorDescription: MSALNativeAuthErrorMessage.unexpectedResponseBody)))
         case .failure(let responseError):
             guard let initiateResponseError = responseError as? MSALNativeAuthSignInInitiateResponseError else {
-                MSALLogger.log(
+                MSALLogger.logPII(
                     level: .error,
                     context: context,
-                    format: "signin/initiate: Unable to decode error response: \(responseError)")
+                    format: "signin/initiate: Unable to decode error response: \(MSALLogMask.maskPII(responseError))")
                 return .error(.unexpectedError(.init(errorDescription: MSALNativeAuthErrorMessage.unexpectedResponseBody)))
```

Comment:
Is there a reason you didn't mark as PII also the log at line 104? I think we should mark that log as PII

Reply:
Done